.pullapprove.yml

```diff
@@ -1,8 +1,10 @@
 approve_by_comment: true
-approve_regex: ^LGTM
+approve_regex: '^(Approved|lgtm|LGTM|:shipit:|:star:|:\+1:|:ship:)'
```

Comment:
i would just stick with `LGTM` it has been the standard way that we have been reviewing PRs for a long time now and thinks like +1 mean something else.